reference/orders.v2.oas2.yml

```diff
@@ -1093,8 +1179,196 @@ paths:
+      summary: List Consignment Shipping Quotes
+      tags:
+        - Order Consignments
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/shippingQuotes_Base'
+              examples:
+                Fixed Shipping:
+                  value:
+                    id: '42'
+                    uuid: 7c65e0a0-cf46-4438-934e-c7b4edb31e18
+                    timestamp: 'Fri, 24 Jun 2022 01:45:27 +0000'
+                    shipping_provider_id: bcstatic
+                    shipping_provider_quote: []
+                    provider_code: peritem
+                    carrier_code: ''
+                    rate_code: ''
+                    rate_id: ''
+                    method_id: 2
+                Fedex:
+                  value:
+                    id: '43'
+                    uuid: 551404b9-beeb-4e49-922b-3b8d85002863
+                    timestamp: 'Fri, 24 Jun 2022 03:52:03 +0000'
+                    shipping_provider_id: bcrealtime
+                    shipping_provider_quote:
+                      rate:
+                        value: '107.32'
+                        unit: USD
+                      transitTime: '1'
+                      name: Priority Overnight
+                      signatureConfirmationFee: {}
+                      carrierName: ''
+                      carrierCode: {}
+                      code: {}
+                      deliveryMessage: ''
+                      labelSizes: []
+                      insuredMailFee: {}
+                      dates: []
+                      rateId: {}
+                      description: ''
+                      additionalInfo: {}
+                    provider_code: fedex
+                    carrier_code: ''
+                    rate_code: ''
+                    rate_id: ''
+                    method_id: 5
+        '404':
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: object
+                  properties:
+                    status:
+                      type: integer
+                    message:
+                      type: string
+              examples:
+                response:
+                  value:
+                    - status: 404
+                      message: The requested resource was not found.
+      operationId: get-orders-orderId-consignments-shipping-shippingId-shippingQuotes
+      description: <p>Gets all shipping quotes persisted on an order for a shipping consignment.</p> <p>This is a read only endpoint and the output can vary based on the shipping quote. A shipping quote can only be generated using the storefront at this time. Orders that are created in the control panel or using the API return a 204 for this endpoint since a shipping quote is not generated during that process.</p>
```

Comment:
```suggestion
      description: |-
        Gets all shipping quotes persisted on an order for a shipping consignment.

        This is a read-only endpoint and the output can vary based on the shipping quote. A shipping quote can only be generated using the storefront. Orders that are created in the control panel or using the API return a `204` for this endpoint since a shipping quote is not generated during that process.
```